### PR TITLE
Update edge-fqm test to match the updated API

### PR DESCRIPTION
**Pull request by mweaver-ebsco**

### CODE REVIEW

**Review by ncovercash — APPROVED** [review]

**Review by kjain110 — APPROVED** [review]

File: edge-fqm/src/main/resources/corsair/edge-fqm/features/edge-entity-types.feature [hunk 1]
```diff
@@ -80,6 +80,6 @@ Feature: Entity types
     And path 'entity-types'
     When method GET
     Then status 200
-    * def baseResponse = response.entityTypes
+    * def baseResponse = response
```
Comment (mweaver-ebsco) [inline edge-fqm/src/main/resources/corsair/edge-fqm/features/edge-entity-types.feature:83]: I don't think this is enough. All this change does is make it so the next line will get tripped up by a new shape of data :P

Comment (ncovercash) [reply edge-fqm/src/main/resources/corsair/edge-fqm/features/edge-entity-types.feature:83]: ah yeah, may want to remove line 84

Comment (ncovercash) [reply edge-fqm/src/main/resources/corsair/edge-fqm/features/edge-entity-types.feature:83]: or change it. i remember something funky about the way the `missingPermissions` was handled (iirc mod-fqm-manager included `missingPermissions: null` and edge didn't?)

Comment (mweaver-ebsco) [reply edge-fqm/src/main/resources/corsair/edge-fqm/features/edge-entity-types.feature]: I'm thinking: go with the simple `And match response == edgeResponse` and see what happens. Worse case, it chokes on the `missingPermissions` and I can fix it easily enough

**Pull request merged**